### PR TITLE
docs: adding ct beta alert box to needed pages

### PR DESCRIPTION
**Pull request by elylucas**

### CODE REVIEW

File: components/global/CtBetaAlert.vue [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<script>
+export default {}
+</script>
+
+<template>
+  <Alert type="warning">
+    <Icon name="exclamation-triangle"></Icon> Component Testing is currently in
+    beta.
```
Comment (amirrustam) [inline components/global/CtBetaAlert.vue:8]: This is sufficient messaging. @cowboy did you think any other details are necessary?

File: content/guides/getting-started/writing-your-first-component-test.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ title: Writing Your First Component Test
 
 </Alert>
 
+<CtBetaAlert></CtBetaAlert>
+
```
Comment (amirrustam) [inline content/guides/getting-started/writing-your-first-component-test.md:18]: To be consistent this banner should go at the very top of the content, so before the "what you'll learn block"

File: components/global/CtBetaAlert.vue (hunk 1 above)
Comment (cowboy) [reply components/global/CtBetaAlert.vue:8]: Seems fine, unless we want to explain _why_ anywhere.

Comment (cowboy) [inline components/global/CtBetaAlert.vue:8]: This seems to need surrounding `<p></p>` tags to render correctly.

```suggestion
    <p><Icon name="exclamation-triangle"></Icon> Component Testing is currently in
    beta.</p>
```

Before

image

After

image

**Review by cowboy — CHANGES_REQUESTED** [review]

**Review by cowboy — APPROVED** [review]

**Review by cowboy — APPROVED** [review]

**Pull request merged**